frontend/.env.example

```
@@ -11,10 +11,21 @@ JWT_SECRET=your-jwt-secret-here
 # 백엔드와 동일한 시크릿을 사용해야 합니다
 QUEUE_TOKEN_SECRET=your-queue-token-secret-here
 
+# PortOne V2 본인인증
+NEXT_PUBLIC_PORTONE_STORE_ID=your-portone-store-id
+NEXT_PUBLIC_PORTONE_CHANNEL_KEY=your-portone-channel-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_PORTONE_CHANNEL_KEY key should go before the NEXT_PUBLIC_PORTONE_STORE_ID key

(UnorderedKey)